### ISSUE TITLE
Sun Blade support - does it appear as removable media ?

### ISSUE THREAD

**Issue by Sch-LikA** [dosdude1/2.5-inch-ide-ssd#3]
First thanks a lot for that design ! I have a Sun Blade 100 workstation on which I tried numerous IDE to sata / CF / msata ... adapters, but none work, only a (any) regular mechanical hdd does. I think it may be due to the fact that those devices appear as removable media (wild guess for now). Does your ide ssd register as a regular "fixed" hdd ?

**Comment by dosdude1** [comment]
Yes, it certainly should. I've had these SSDs work with many machines that are picky about their hard disks.
